src/vs/workbench/contrib/notebook/browser/viewModel/foldingModel.ts

```diff
@@ -319,3 +312,14 @@ export function updateFoldingStateAtIndex(foldingModel: FoldingModel, index: num
 	const range = foldingModel.regions.findRange(index + 1);
 	foldingModel.setCollapsed(range, collapsed);
 }
+
+export function* getMarkdownHeadersInCell(cellContent: string): Iterable<{ readonly depth: number; readonly text: string }> {
```

Comment:
Why use a generator if you are gonna spread the result into an array?

Reply:
Should be used directly in a `for..of` now